Vostok.ClusterClient.Transport.Tests/Helpers/SocketTestServer.cs

```diff
@@ -36,9 +37,13 @@ public static SocketTestServer StartNew(string response)
 
         public string LastRequest { get; private set; }
 
-        public void Dispose() => listener.Stop();
+        public void Dispose()
+        {
+            listener.Server.Shutdown(SocketShutdown.Both);
```

Comment:
On my Windows machine this line always throw exception:
```
System.Net.Sockets.SocketException : A request to send or receive data was disallowed because the socket is not connected and (when sending on a datagram socket using a sendto call) no address was supplied.
   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, String callerName)
   at System.Net.Sockets.Socket.Shutdown(SocketShutdown how)
   at Vostok.Clusterclient.Transport.Tests.Helpers.SocketTestServer.Dispose() in C:\src\service-discovery\vostok.clusterclient.transport\Vostok.ClusterClient.Transport.Tests\Helpers\SocketTestServer.cs:line 42
   at Vostok.Clusterclient.Transport.Tests.Functional.Common.SendFailureTests`1.Should_return_SendFailure_code_when_socket_closed_while_sending_large_body() in C:\src\service-discovery\vostok.clusterclient.transport\Vostok.ClusterClient.Transport.Tests\Functional\Common\SendFailureTests.cs:line 33
```

But without this line all tests passed.

Reply:
fixed with try catch, really don't want to guess in which environment we need this line and in which don't